### ISSUE TITLE
Syntax definition error on Function Foward sentences

### ISSUE THREAD

**Issue by Banegasn** [jcaillon/3P#144]
Example:

FUNCTION example RETURNS char (input d as date, input t as int) FORWARD. 

This line should not have a -/+ sign to expand and collapse, its a unique line statement

![example](https://cloud.githubusercontent.com/assets/5572396/21855597/c0a7ac78-d7fd-11e6-86c3-bd175b986774.png)

**Comment by jcaillon** [comment]
Hi,

Thanks for reporting the bug. However, I will not likely solve it soon. At the moment, I'm using the built in "User defined language" of N++ but this system is limited (the problem you are pointing exists because of that limitation).

The only way around this is to build a custom scintilla lexer in c or c++, but since I already built a lexer in C# i don't really want to do that. I was close to finding a solution based purely on scintilla API (that i can call through the C# plugin) but there were performance issues and I didn't pursue this lead further because I was lacking time...

TL;DR : I'm aware of the problem, might solve it one day, but it takes a lot of time
